packages/form-layout/src/vaadin-form-layout-styles.js

```diff
@@ -84,8 +84,22 @@ export const formLayoutStyles = css`
   }
 
   :host([auto-responsive][auto-rows]) #layout ::slotted(*) {
+    grid-column-start: var(--_column-start, auto);
+  }
+`;
+
+export const formRowStyles = css`
+  :host {
+    display: contents;
+  }
```

Comment:
Let's also add `:host([hidden])` styles to this component.

Reply:
Thanks, done.